File: disciplinas-faculdade/primeiro-periodo/residencia-tecnologica/introduçãoEmC/5.c
#include <stdio.h>
#include <stdlib.h>

float num;

int main () {
	printf("Digite um numero em centimetros: ");
	scanf("%f",&num);
	
	printf("%.2f e igual a %.2f polegadas",num, num / 2.54);
	
	return 0;
};

File: disciplinas-faculdade/primeiro-periodo/logica-e-algoritmo/08.06.23/5.js
function verFatorial(numero){
    const num = numero

    if (numero === 0 || numero === 1){
        return 1;
    } else {
        let resultado = numero;
        while (numero > 1){
            resultado *= (numero - 1)
            numero--
        }
        console.log(`${num}! = ${resultado}`)
    }
}

verFatorial(5)
